tests/foreman/cli/test_oscap.py

```diff
@@ -104,8 +104,9 @@ def setUpClass(cls):
             remote_file="/tmp/{0}".format(cls.file_name)
         )
         cls.title = 'rhel-6-content'
-        result = Scapcontent.info({'title': cls.title})
-        if not result['title'] in cls.title:
+        result = [scap['title'] for scap in Scapcontent.list() if
+                  scap.get('title') in cls.title]
+        if cls.title not in result:
```

Comment:
Think there is some confusion here, I see the code more cleaner like this
```python
   result = [scap['title'] for scap in result if
                   cls.title in scap['title']]
    if not result:
        # create a new one
```

Reply:
it can return more than , one element in the list due to https://bugzilla.redhat.com/show_bug.cgi?id=1474172

Reply:
I can undestand that like this: 
```
if scap['title'] is in cls.title 
and 
cls.title is in scap['title'] 
=> scap['title'] == cls.title
```